yii2/config/db.php

```diff
@@ -2,8 +2,8 @@
 
 return [
     'class' => 'yii\db\Connection',
-    'dsn' => 'mysql:host=localhost;dbname=yii2basic',
+    'dsn' => 'mysql:host=localhost;dbname=dz5',
     'username' => 'root',
-    'password' => '',
+    'password' => '1111',
```

Comment:
don't commit your passwords here. Right now leave this configuration as is, but in future you can add this file (or other configuration file) to .gitignore